tests/unittest_nodes_position.py

```diff
@@ -0,0 +1,152 @@
+import textwrap
+from typing import List
+
+from astroid import builder, nodes
+
+
+class TestNodePosition:
+    """Test node ``position`` attribute."""
+
+    @staticmethod
+    def test_position_class() -> None:
+        """Position should only include keyword and name.
+
+        >>> class A(Parent):
+        >>> ^^^^^^^
+        """
+        code = textwrap.dedent(
+            """
+        class A:  #@
+            ...
+
+        class B(A):  #@
+            pass
+
+        class C:  #@
+            '''Docstring'''
+
+            class D:  #@
+                ...
+
+        class E:  #@
+            def f():
+                ...
+        """
+        ).strip()
+        ast_nodes: List[nodes.NodeNG] = builder.extract_node(code)  # type: ignore
```

Comment:
I was thinking of turning on https://github.com/python/mypy/pull/11633 when it gets released.
Could you add the error here?

Reply:
That looks like a great idea! Looking forward to it.